### ISSUE TITLE
"Recently added flights" aren't recent

### ISSUE THREAD

**Issue by reedy** [jpatokal/openflights#1225]
The homepage (when logged out), has a title of "Recently added flights", which would be a nice feature if it actually worked.

In reality, it seems to be a snapshot of some random (test data?) flights.

Simplest fix is to maybe just remove/disable this title, for now at least?

![Screenshot 2023-07-07 at 14 36 32](https://github.com/jpatokal/openflights/assets/67615/d25ad0d8-6cbe-41c0-aa09-0acc943e28dc)

**Comment by chrisrosset** [comment]
The data is returned from `map.php` which [uses user ID = 1](https://github.com/jpatokal/openflights/blob/ecf635a4bc460a5ce3a911c34305672188ad6d80/php/map.php#L33) if you're not logged in. So this is whatever is defined in the database for that user.

This, just like another issue you reported (#1136), appears to be a case of a missing cron job. This data is supposed to be updated by [`update-demo.sql`](https://github.com/jpatokal/openflights/blob/ecf635a4bc460a5ce3a911c34305672188ad6d80/sql/update-demo.sql).

**Comment by reedy** [comment]
> This, just like another issue you reported (#1136), appears to be a case of a missing cron job. This data is supposed to be updated by [`update-demo.sql`](https://github.com/jpatokal/openflights/blob/ecf635a4bc460a5ce3a911c34305672188ad6d80/sql/update-demo.sql).

I didn't check all the airports, but it does look like that's not been run in a long time...

**Comment by chrisrosset** [comment]
Probably since the server upgrade last year.

**Comment by reedy** [comment]
> The data is returned from `map.php` which [uses user ID = 1](https://github.com/jpatokal/openflights/blob/ecf635a4bc460a5ce3a911c34305672188ad6d80/php/map.php#L33) if you're not logged in. So this is whatever is defined in the database for that user.

That, is, I guess another bug...

I think user ID 1 should be reserved, or some account created as part of the install.

On my dev wiki, my account is ID 1 (of course), so it's the recent flights too...

Probably relevant to your docker setup too in #1206 :)

**Comment by chrisrosset** [comment]
> On my dev wiki, my account is ID 1 (of course), so it's the recent flights too...

Yeah, this is exactly how I found out too. I was making changes to the map and noticed that it didn't change when logging in/out. It freaked me out - I thought I broke it. :)

**Comment by reedy** [comment]
![Screenshot 2023-07-11 at 00 50 11](https://github.com/jpatokal/openflights/assets/67615/d22489ec-a9be-4e52-81ea-f737d931607c)

**Comment by chrisrosset** [comment]
I just found https://github.com/jpatokal/openflights/issues/646#issue-197200991 which includes a 2016 screenshot with that 2030-01-01 WAW-WAW flight we all know and love.
